lang/en.json

```diff
@@ -170,8 +172,8 @@
         "DomReadyTriggerName": "DOM Ready",
         "DriftDescription": "Drift allows you to connect with your customers.",
         "DriftHelp": "This tag allows you to add the Drift contact form to your website.",
-        "AllDownloadsClickTriggerDescription": "Triggered when a link is clicked which links to a downloadable file.",
-        "AllDownloadsClickTriggerHelp": "Triggered when a user clicks on an \"A\" or \"AREA\" element and the link points to a file with a downloadable file extension.",
+        "AllDownloadsClickTriggerDescription": "Triggered when a link is clicked which links to a downloadable file. It will be triggered on left,middle and right click.",
```

Comment:
there's a space missing after the comma @AltamashShaikh

Reply:
same also for the others below

Reply:
Updated